takeshape/penny
# Provide an account activation page

### Test Plan (Steps to test):

1. Perform a password recovery (forgot my password) on an active account
1. Click the Shopify recovery link in the email you are sent
1. Copy the full path and append to the URL for this PR. Hit enter.
2. Attempt to reset your password. It should succeed, notify you and redirect to the sign in page.
3. Do all of the steps above, but with a known inactive account. (#172 will need to merge first)
